### ISSUE TITLE
checkout/pull should write a summary of the working space changes

### ISSUE THREAD

**Issue by shcheklein** [iterative/dvc#1838]
Now it writes the same message:

```
Preparing to collect status from https://remote.dvc.org/get-started
[##############################] 100% Collecting information
[##############################] 100% Analysing status.
[##############################] 100% Checkout finished!

Everything is up to date.
```

no matter if something changed or not. It's hard to see if it actually update the working space. I would suggest to write summary, similar to the way git is doing when you pull/push something.

**Comment by shcheklein** [comment]
Now w/o a progress bar it's even worse - `Everything is up to date.` for `dvc pull` and an empty output for the `dvc checkout`. I even though for a moment that I didn't know how `dvc pull` works and went to check the docs, before I realized that something is actually happening, it's changing something.

**Comment by jorgeorpinel** [comment]
Seems like both @bhavaniravi and @leehanchung also reported this in #2667 and #2838, respectively. Their comments:

> When I do a dev pull and the data is fetched I don't get any response saying Sucess I have do a ls to verify whether the data folder has been created

---

> ```powershell
> (base) PS B:\user\example-get-started> ls .\data\
> 
>     Directory: B:\user\example-get-started\data
> 
> Mode                LastWriteTime         Length Name
> ----                -------------         ------ ----
> -a----       11/22/2019  12:52 PM             11 .gitignore
> -a----       11/22/2019  12:52 PM            155 data.xml.dvc
> 
> (base) PS B:\user\example-get-started> dvc pull
> Everything is up to date.
> (base) PS B:\user\example-get-started> ls data
> 
>     Directory: B:\user\example-get-started\data
> Mode                LastWriteTime         Length Name
> ----                -------------         ------ ----
> -a----       11/22/2019  12:52 PM             11 .gitignore
> -a----       11/22/2019   2:52 PM       37916850 data.xml
> -a----       11/22/2019  12:52 PM            155 data.xml.dvc
> ```
> basically it doesnt tell what is pulled/changed at the moment.
> And i do see a point of doing it none-verbose-ly if the dataset contains thousands of images inside a directory. :p

**Comment by casperdcl** [comment]
so should #2667 be closed as a duplicate?

**Comment by jorgeorpinel** [comment]
Idk. Already asked over there. Ivan and Ruslan have a long conversation in there, so let them decide.

**Comment by efiop** [comment]
Closed #2667 in favor of this one. Thanks for the heads up!

**Comment by efiop** [comment]
Fixed by https://github.com/iterative/dvc.org/pull/1054
